common/src/main/scala/uk/ac/wellcome/platform/stacks/common/models/Identifier.scala

```diff
@@ -1,27 +1,22 @@
 package uk.ac.wellcome.platform.stacks.common.models
 
-import uk.ac.wellcome.platform.sierra.models.Hold
-
 import scala.util.{Failure, Success, Try}
 
 sealed trait Identifier[T] {
   val value: T
+
+  override def toString: String = value.toString
 }
 
 case class SierraItemIdentifier(value: Long) extends Identifier[Long]
 
 object SierraItemIdentifier {
   def createFromString(id: String): SierraItemIdentifier = Try {
-    id.toLong
+    id.split("/").last.toLong
```

Comment:
Can you provide an example of what you expect this string to look like?

I think it’s `sierra-system-number/1234567`, but I could be wrong. It’ll make it obvious if this ever diverges from the catalogue code.

Reply:
Edit: I see this mirrors the `createFromHold` method, and I have no idea what that method was meant to be.